front/app/containers/IdeasNewSurveyPage/IdeasNewSurveyForm/index.tsx

```diff
@@ -188,10 +190,23 @@ const IdeasNewSurveyForm = ({ project, phaseId }: Props) => {
       publication_status: data.publication_status || 'published',
     };
 
+    const handleOnError = () => {
+      // If an error happens, it's likely some permission issues.
+      // We refetch the project to use the correct action descriptors.
+      queryClient.invalidateQueries({
+        queryKey: projectsKeys.all(),
+      });
+    };
     // Update or add the idea depending on if we have an existing draft idea
     const idea = ideaId
-      ? await updateIdea({ id: ideaId, requestBody })
-      : await addIdea(requestBody);
+      ? await updateIdea(
+          { id: ideaId, requestBody },
+          {
+            onError: handleOnError,
+          }
+        )
+      : await addIdea(requestBody, { onError: handleOnError });
```

Comment:
Strictly speaking, it's not used atm. But it can be used later if we change permissions when creating ideas.